src/test/java/kitchenpos/repository/MenuRepositoryTest.java

```diff
@@ -116,12 +109,14 @@ public MenuRepositoryTest(final MenuRepository menuRepository) {
                 () -> assertThat(menus.get(0))
                         .usingRecursiveComparison()
                         .ignoringFields("price", "menuProducts")
-                        .isEqualTo(new Menu(
-                                1L,
-                                "후라이드치킨",
-                                new Price(BigDecimal.valueOf(16000)),
-                                2L,
-                                new ArrayList<>()
-                        )));
+                        .isEqualTo(menu(1L, "후라이드치킨", 16000)));
+    }
+
+    private Menu menu(final long id, final String name, final int price) {
+        return new Menu(id, name, price(price), 2L, new ArrayList<>());
+    }
+
+    private Price price(final int price) {
+        return new Price(BigDecimal.valueOf(price));
```

Comment:
메서드명은 동사인게 좋을 것 같아요.

```suggestion
    private Menu createMenu(final long id, final String name, final int price) {
        return new Menu(id, name, price(price), 2L, new ArrayList<>());
    }

    private Price createPrice(final int price) {
        return new Price(BigDecimal.valueOf(price));
```